streamlit_web.py

```diff
@@ -651,6 +654,49 @@ def interactive():
         summaries = sz.summarize_text(input_text)
         st.write(summaries)
 
+def senti_pos_iter(tokens_column):
+    # Start off with an empty list
+    display_series = []
+    # For each item in the already existing tokens column in the data frame,
+    for token_element in tokens_column:
+        # Start off with an empty words list to add to over time
+        words = []
+        # If the word isn't in the words list already, then it's added to the words list
+        for word in token_element:
+            if word not in words:
+                words.append(word)
+        # Insertion sort algorithm using the polarity to compare the words
+        for i in range(len(words)):
+            key = words[i]
+            j = i - 1
+            while j >= 0 and TextBlob(words[j]).sentiment.polarity > TextBlob(key).sentiment.polarity:
+                words[j + 1] = words[j]
+                j -= 1
+            words[j + 1] = key
```

Comment:
This is the same functionality as the `create_word_list` in `analyzer.py` right?